client/modules/PostEditor/PostEditor.jsx

```diff
@@ -56,6 +56,7 @@ function PostEditor(props: types.Props) {
   );
 
   const [categoryId, setCategoryId] = useState(null);
+  /* eslint-disable no-unused-vars */
```

Comment:
You've removed it from usage, but it marks form as dirty - `setIsDirty` - you need to apply this functionality also on your dropdown.